dynamiqs/smesolve/sme_solver.py

```diff
@@ -79,37 +82,40 @@ def sample_wiener(self, dt: float) -> Tensor:
         ).to(dtype=self.rdtype)
 
     @cache
-    def Lp(self, rho: Tensor) -> Tensor:
-        # rho: (b_H, b_rho, ntrajs, n, n) -> (b_H, b_rho, ntrajs, n, n)
-        # L @ rho + rho @ Ldag
-        L_rho = torch.einsum('bij,...jk->...bik', self.meas_ops, rho)
-        return L_rho + L_rho.mH
+    def Vp(self, rho: Tensor) -> Tensor:
+        # rho: (b_H, b_rho, ntrajs, n, n) -> (b_H, b_rho, ntrajs, len(V), n, n)
+        # V @ rho + rho @ Vdag
+        V_rho = torch.einsum('bij,...jk->...bik', self.V, rho)
```

Comment:
Probably faster to use something else than einsum when possible. Here I'm guessing a simple `@` is best.

Reply:
`@` doesn't work (`self.V` has shape `(len(V), n, n)` and `rho` has shape `(..., n, n)`).